server/src/util.ts

```diff
@@ -58,7 +58,7 @@ export async function getDownloadURL(): Promise<string> {
   return `https://binaries.prisma.sh/master/${version}/${platform}/prisma-fmt${extension}`
 }
 
-export function getCLIVersion(packageName: string) {
+export function getCLIVersion(packageName: string): Buffer {
   const fileName =
     packageName === 'prisma-dev'
       ? 'prisma_version_unstable'
```

Comment:
Can you rely on reading `package.json` file for getting the actual CLI version? Just wrt separation of code. I expect scripts part of code to never be referred by the main extension/LSP code.

Reply:
I agree, but also I don't know any other way to get the actual CLI version other than the package name..

Reply:
Just reading it from `server/package.json` (one of the Prisma packages) should suffice. In an actual build of the extension, it will have the correct value.

Reply:
Do you mean in `version` in `server/package.json`? For startup, I'm getting `version` already, but the value is different: 0.0.1245 instead of 2.0.0-alpha-1245. Is there any way I can get the ladder one?